lib/sudo-exec/src/lib.rs

```diff
@@ -80,15 +79,16 @@ pub fn run_command(ctx: Context, env: Environment) -> io::Result<(ExitReason, im
     let monitor_pid = fork()?;
     // Monitor logic. Based on `exec_monitor`.
     if monitor_pid == 0 {
-        match monitor::MonitorRelay::new(command, pty_follower, backchannels.monitor)?.run() {}
+        let (mc, mut ev) = monitor::MonitorRelay::new(command, pty_follower, backchannels.monitor);
```

Comment:
longer variable names would help readability here. I though the first element of the tuple was the `MonitorRelay` so why is it shortened to `mc`?

same comment for the `PtyRelay` below

Reply:
I'm reading too much C :sweat_smile: